README.md

```diff
@@ -11,8 +11,7 @@ The lexer is registered globally under the name `ansible-output`.
 It also provides a [Pygments] style for tools needing to highlight
 code snippets.
 
-Unless otherwise noted in the code, it is licensed under the terms
-of the GNU General Public License v3 or, at your option, later.
+The code is licensed under the terms of the BSD 2-Clause license.
```

Comment:
nit: I like external things to be interlinked

```suggestion
The code is licensed under the terms of the [BSD 2-Clause license].

[BSD 2-Clause license]: https://opensource.org/licenses/BSD-2-Clause
```

Reply:
Addressed in 3863fd121a5879021fce5d13af5d34ea5f8cc45d.